Testscripts/Linux/KVP-BasicChecks.sh

```diff
@@ -31,7 +31,7 @@ fi
 
 if [ "${kvp_items:-UNDEFINED}" = "UNDEFINED" ]; then
     LogErr "The test parameter kvp_items is not defined in constants.sh"
-    SetTestStateAborted
+    SetTestStateSkipped
```

Comment:
This approach is not very good.

Lines 23-36 must be moved after the check if KVP daemon is running (and don't skip there).

Current line 44 that does the kvp daemon check must exit with Skip if it doesn't find it running.

Reply:
@chvalean thanks, Lines 23-36 do the check values from constants.sh file, put it before or after check kvp, does it matter?

Reply:
Ah, understood.